src/App.js

```diff
@@ -1,19 +1,23 @@
 import React from 'react';
 import TodoList from './TodoList';
 import AddTodoForm from './AddtodoForm';
+import "./style.css"
```

Comment:
You only need to import styles where you using them. For example, if you imported a style to use on line 16 you should import it. Here we are using not using any styles from the styles.css file.